toolchain/check/decl_name_stack.h

```diff
@@ -153,13 +153,13 @@ class DeclNameStack {
       // expressions. `None` indicates resolution failed.
       SemIR::InstId resolved_inst_id;
 
-      // The ID of an unresolved identifier.
-      SemIR::NameId unresolved_name_id = SemIR::NameId::None;
+      // When `state` is `Poisoned` (name is unresolved due to name poisoning),
+      // the poisoning location.
+      SemIR::LocId poisoning_loc_id = SemIR::LocId::None;
     };
 
-    // When `state` is `Poisoned` (name is unresolved due to name poisoning),
-    // the poisoning location.
-    SemIR::LocId poisoning_loc_id = SemIR::LocId::None;
+    // The ID of an identifier.
```

Comment:
```suggestion
    // The most recently applied name.
```

Reply:
It seems you missed this comment, so I've created #4977 to improve this comment.

Reply:
Indeed missed this one, sorry about that!
Thanks for sending the PR.